talisker/gunicorn.py

```diff
@@ -90,6 +89,9 @@ def get_extra(self, resp, req, environ, request_time):
         if request_id:
             extra['request_id'] = request_id
 
+        msg = "{} {}".format(extra['method'], extra['path'])
+        if extra['qs']:
+            msg += '?'
```

Comment:
The '?' is just an indicator there's also a query string to look at, right? Do you think it's worth including? If so, maybe it's worth bundling in the `format` call above to avoid the extra string append.